SteamKit2/SteamKit2/Steam/Authentication/SteamAuthentication.cs

```diff
@@ -139,6 +139,14 @@ public async Task<CredentialsAuthSession> BeginAuthSessionViaCredentialsAsync( A
 
             // Encrypt the password
             var publicKey = await GetPasswordRSAPublicKeyAsync( details.Username! ).ConfigureAwait( false );
+
+            // Password limit is 64. If it's longer, trim it.
+            if (!string.IsNullOrEmpty(details.Password) && details.Password.Length > 64)
+            {
+                DebugLog.WriteLine("SteamAuthentication", "Notice: password is longer than 64 characters and will be trimmed.");
```

Comment:
```suggestion
                DebugLog.WriteLine(nameof(SteamAuthentication), "Notice: password is longer than 64 characters and will be trimmed.");
```

Make a const variable for the "64".

Also, you didn't need to make a new pull request, you could have just pushed to the same branch.